connectors/src/connectors/github/temporal/client.ts

```diff
@@ -67,13 +65,7 @@ export async function launchGithubFullSyncWorkflow({
   }
 
   await client.workflow.start(githubFullSyncWorkflow, {
-    args: [
-      dataSourceConfig,
-      connectionId,
-      connectorId,
-      syncCodeOnly,
-      forceCodeResync,
-    ],
+    args: [dataSourceConfig, connectorId, syncCodeOnly, forceCodeResync],
```

Comment:
So we're changing the number of params ? How will this work ?

Reply:
This will fail I'll cancel :+1: